Packages/vcs/vcs/utils.py

```diff
@@ -312,42 +312,6 @@ def _scriptrun(script, canvas=None):
                 vcs.process_src_element(s)
                 processing = False
     f.close()
-    # Ok now we need to double check the isolines
-    gd = vcs.elements["isoline"]["default"]
-    for g in vcs.elements["isoline"].values():
-        if g.name == "default":
-            continue
-        for att in ["line", "textcolors", "text"]:
-            try:
-                setattr(g, att, getattr(g, att))
-            except:
-                lst = []
-                if att == "line":
-                    for e in g.line:
-                        if e in vcs.elements["line"]:
-                            lst.append(vcs.elements["line"][e])
-                        else:
-                            lst.append(e)
-                elif att == "text":
-                    for e in g.line:
-                        if e in vcs.elements["textorientation"]:
```

Comment:
because textorientation objects contain the relevant information to this attribute so it is possible that the use rsent us a list of textorientation to look for. We could probably drop this and the `texttable` bellow and simply look in `textcombined` it would be easier to understand and maintain.